suse_caasp

```diff
@@ -257,6 +257,31 @@ rpm_verify() {
         fi
 }
 
+# Input logfilename containername command
+docker_exec() {
```

Comment:
This function makes always use of `entrypoint.sh` which is available only inside of the velum image. I would change the name of this function to make explicit it can be used to debug the velum containers only.